catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/QueryMetacardApplication.java

```diff
@@ -98,6 +104,14 @@ public void init() {
           int start = getOrDefaultParam(req, START, MIN_START);
           int count = getOrDefaultParam(req, COUNT, MAX_PAGE_SIZE);
 
+          String attr = req.queryParams(ATTR);
+
+          if (attr == null) {
```

Comment:
❓ Not sure how this works, do we also need to use default if empty or invalid attribute? Might go out of scope for that second one.

Reply:
As it currently exists this endpoint returns all searches in descending order based on their last modified attribute. 

If there are preferred defaults we will find them as we iterate. 

Modified the code to handle blank and invalid entries